src/pkg/message/progress.go

```diff
@@ -40,6 +40,20 @@ func NewProgressBar(total int64, text string) *ProgressBar {
 	}
 }
 
+// Close doesn't do anything but satisfy implementation.
+// https://github.com/defenseunicorns/pkg/blob/325e69d2560e8d767511c9e27187f775ecbad3fa/helpers/progress.go#L31-L38
+func (*ProgressBar) Close() error {
```

Comment:
Instead of putting comments on these functions linking to code, I would either just delete them or explain why the implementation is unnecessary

Reply:
implemented Close in favor of Stop